src/main/java/org/snomed/snowstorm/ecl/domain/expressionconstraint/SSubExpressionConstraint.java

```diff
@@ -339,9 +339,18 @@ private Set<Long> applyConceptCriteriaWithOperator(Collection<Long> conceptIds,
                                     .should(termsQuery(QueryConcept.Fields.CONCEPT_ID, retrieveAllAncestors(conceptIds, branchCriteria, stated, conceptSelector)))
                                     .should(termsQuery(QueryConcept.Fields.CONCEPT_ID, conceptIds)))
                     );
-            case memberOf ->
-                // ^
-				queryBuilder.must(termsQuery(QueryConcept.Fields.REFSETS, conceptIds));
+            case memberOf -> {
```

Comment:
This change fixes ECLQueryServiceFilterTest.testMemberFieldFilter which was previously disabled. This simply falls back to the non-perf code in the main Snowstorm fork if there are memberFilterConstraints, but if there are none will use the performance enhancement.